test/test_transforms_v2_refactored.py

```diff
@@ -3009,3 +3013,104 @@ def test_auto_augment_policy_error(self):
     def test_aug_mix_severity_error(self, severity):
         with pytest.raises(ValueError, match="severity must be between"):
             transforms.AugMix(severity=severity)
+
+
+class TestConvertBoundingBoxFormat:
+    old_new_formats = pytest.mark.parametrize(
+        ("old_format", "new_format"), list(itertools.product(iter(tv_tensors.BoundingBoxFormat), repeat=2))
+    )
+
+    @old_new_formats
```

Comment:
It is easier to understand what this does by using the same logic as `cpu_and_cuda()`, i.e. just let `old_new_format` be an iterable instead of a parametrization:

```py
@pytest.mark.parametrize("old_format, new_format", old_new_formats)
```


When one reads `old_new_formats`, it's not obvious at all that this is a parametrization without having to jump up to the definition.